src/assets/styles/base/ngx-modal.scss

```diff
@@ -0,0 +1,13 @@
+// ngx smart modal
+.nsm-content {
+  padding: 0 !important;
+}
+
+.nsm-large {
```

Comment:
I know this sometimes is hard, but we should try to reserve the use of !important. sometimes this can simply be fixed by re-arranging css, sometimes not so much.... Having said this if it works, leave it alone...

Reply:
For context, this is a 3rd-party library I'm trying to modify so there's an added stickiness to their CSS. That said, I think I managed to solve this without important and left it in. I'll look back at it.